test/Frame.spec.jsx

```diff
@@ -355,4 +355,15 @@ describe('The Frame Component', () => {
     div.remove();
     ReactDOM.render(<Frame />, div);
   });
+
+  it('should not error when root component is re-appended', () => {
```

Comment:
When you say "not error" - it's that second `render` which would throw without this patch, right?

Reply:
yup - its the `re` append that does it.